tests/run_tests/getTestsToIgnore.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const path = require('path');
+
+const ignoredTestsPath = '.github/workflows/ignore_tests/';
+
+function getTestNames() {
+  const files = fs.readdirSync(ignoredTestsPath);
+  return files.reduce((names, file) => {
+    const content = fs.readFileSync(path.join(ignoredTestsPath, file), 'utf-8');
+    return names + content;
+  }, '');
+}
+
+function replacer(testNames) {
+  return testNames.replace(/\n(?=[^\n]*\n)/g, '|').replace('\n', '');
+}
+
+module.exports = () => {
+  return replacer(getTestNames());
+};
```

Comment:
Instead of reading the whole file then replace the line breaks with `|` and removing the last empty line, what do you think about reading the file line by line, add them to an array (besides the empty ones), and then use `array.join('|')`? You can do this either with await or using the stream directly (using the `line` event) https://nodejs.org/api/readline.html#readline_example_read_file_stream_line_by_line

This would imply that you would have to wait for this async process to finishing runTests.js and when it's over you can then call `mocha.run` inside the callback.